new-package/elm-review-package-tests/check-previews-compile.js

```diff
@@ -1,16 +1,18 @@
 #!/usr/bin/env node
 
-const path = require('path');
+const path = require('node:path');
 const Ansi = require('./helpers/ansi');
-const {execSync} = require('child_process');
+const {execSync} = require('node:child_process');
 const {findPreviewConfigurations} = require('./helpers/find-configurations');
+// @ts-ignore - Generated file.
```

Comment:
Here's another ignore comment. I figure we can call `.replace()` on the file and delete this line.

Reply:
Did it!

Reply:
Not optimal in terms of performance, but I guess it works :sweat_smile:

Reply:
I figure that if we go the route of hardcoding paths for #147, then this wouldn't be so bad 🤷‍♂️
Then again, that's not a performance optimal route...

Reply:
Yes, not an issue in terms of performance. But I'm not sure I get what the relation with #147 is, sorry

Reply:
I thought you meant that we were copying the files, then editing them, but with #147, we'll move them one by one, so we could cut out the duplication, but at the expense of total speed (as a native cp is certainly faster than our kinda-pure-js one would be.)

Reply:
That is what I meant. Even with #147 and specifying paths we still would do this copy+edit+write process for this file, no?

Actually, can't we add a dummy `new-package/elm.json` file to get rid of the TS error entirely?

Reply:
Yeah, I guess we're just talking past each other.
We could. Would our actual one overwrite it?

Reply:
I think that if we get in #147 then we can skip copying the dummy `elm.json`, so maybe it should happen in tandem with #147 (which btw, feel free to take over, I don't remember the state of it).